src/bitflags_trait.rs

```diff
@@ -0,0 +1,46 @@
+use core as _core;
```

Comment:
This is only necessary for use in macros, so that types get resolved correctly. For the definition of the trait, you can just write `core::option::Option` etc.